client/my-sites/importer/newsletter/subscribers/step-initial.tsx

```diff
@@ -35,17 +39,34 @@ export default function StepInitial( {
 		}
 		prevInProgress.current = importSelector?.inProgress;
 	}, [ importSelector?.inProgress, setAutoFetchData ] );
+	const isJetpack = useSelector( ( state ) => isJetpackSite( state, selectedSite.ID ) );
 
 	return (
 		<Card>
 			<h2>{ __( 'Step 1: Export your subscribers from Substack' ) }</h2>
 			<p>
 				{ createInterpolateElement(
 					__(
-						'Generate a CSV file of all your Substack subscribers. On Substack, go to the <strong>Subscribers</strong> tab and click the <strong>Export</strong> button you’ll find on top of the table. Then, upload the downloaded CSV in the next step.'
+						'Generate a CSV of your Substack subscribers. On the free plan, <supportLink>you can import up to 100 subscribers.</supportLink> In Substack, go to <strong>Subscribers</strong>, click <strong>Export</strong> under "All subscribers," then upload the CSV in the next step.'
 					),
 					{
 						strong: <strong />,
+						supportLink: isJetpack ? (
```

Comment:
The Substack importer only exists in WordPress.com right now, so we could keep the behavior the same for both site types. Otherwise, this adds the external link correctly:

<img width="758" alt="Screen Shot 2025-02-24 at 3 42 22 PM" src="https://github.com/user-attachments/assets/d716014e-5de8-46be-9053-b07308d4021f" />